common/dex-auth/base/dex_params.env

```diff
@@ -0,0 +1,21 @@
+# Dex Server Parameters (some params are shared with client)
+dex_domain=dex.example.com
+issuer=https://dex.example.com:32000
+ldap_host=ldap.auth.svc.cluster.local:389
+dex_client_id=ldapdexapp
+oidc_redirect_uris=['http://login.example.org:5555/callback/onprem-cluster']
+dex_application_secret=pUBnBOY80SnXgjibTYM9ZWNzY2xreNGQok
+jwks_uri=https://dex.example.com:32000/keys
+
+# Dex Login Client Parameters
+cluster_name=onprem-cluster
+dex_client_redirect_uri=http://login.example.org:5555/callback/onprem-cluster
+k8s_master_uri=https://k8s.example.com:443
+dex_client_listen_addr=http://127.0.0.1:5555 # Set to HTTPS if TLS is configured
+
+#TODO(krishnadurai): Paramerize certificates:
+# 1. data/ca.pem in ca configMap
+# 2. k8s_ca_pem in dex-k8s-authenticator-cm:cluster
+# 3. idp_ca_pem in dex-k8s-authenticator-cm:cluster
+# Not implemented since multiline values are not getting parsed in kustomize
+# environment file
```

Comment:
Certs need to be parameterized.

@kkasravi any suggestions here? Multi-line values in this file fail.
(Refer this link for details)[https://stackoverflow.com/questions/57142291/multiline-values-in-kustomize-environment-source-file]
This is required since these services can not mount these certs as a Kubernetes secret.

Reply:
Found the solution to this problem using `files` directive for kustomize.